ruby/reek/default.yml

```diff
@@ -55,8 +55,9 @@ detectors:
   ManualDispatch:
     enabled: true
     exclude: []
+  # Not really useful and ignored a lot.
   MissingSafeMethod:
-    enabled: true
+    enabled: false
```

Comment:
bang means that the method is the dangerous version and that a safe version exist.
bang mean that there is a preferable and less dangerous version of the method.
Instead of remove this metric, just stop to use bang.

`concat` is a good example in ruby. It modify the receiver like `map!` or other methods with bang.
It is name `concat` and not `concat!`. There is not link between the modification of the receiver and the bang.

https://www.rubydoc.info/github/troessner/reek/Reek/Smells/PrimaDonnaMethod

Reply:
I know but this is not followed at all in our codebase. It currently makes noise everywhere.